pyrmq/consumer.py

```diff
@@ -77,14 +84,36 @@ def __init__(
             retry_delay=self.retry_delay,
         )
 
+        self.retry_queue_name = f"{self.queue_name}.{self.retry_queue_suffix}"
+
+        queue_args = {
+            "x-dead-letter-exchange": self.exchange_name,
+            "x-dead-letter-routing-key": self.routing_key,
+        }
+
+        self.publisher = Publisher(
```

Comment:
Rename it so that it's clear that it's for retry.

Also, initialize it only if retry is enabled.